stock_request/models/stock_request.py

```diff
@@ -5,25 +5,20 @@
 from odoo.exceptions import UserError, ValidationError
 from odoo.tools import float_compare
 
-REQUEST_STATES = [
-    ("draft", "Draft"),
-    ("open", "In progress"),
-    ("done", "Done"),
-    ("cancel", "Cancelled"),
-]
-
 
 class StockRequest(models.Model):
     _name = "stock.request"
     _description = "Stock Request"
     _inherit = "stock.request.abstract"
     _order = "id desc"
 
-    def __get_request_states(self):
-        return REQUEST_STATES
-
-    def _get_request_states(self):
-        return self.__get_request_states()
+    def _get_states(self):
+        return [
+            ("draft", _("Draft")),
```

Comment:
You cannot include translations like that in selections. I have to check, but I've solved this in another way